static/app/components/demo/demoHeader.tsx

```diff
@@ -1,23 +1,48 @@
+import {useEffect, useState} from 'react';
 import styled from '@emotion/styled';
 
 import Button from 'app/components/button';
 import ButtonBar from 'app/components/buttonBar';
 import ExternalLink from 'app/components/links/externalLink';
 import LogoSentry from 'app/components/logoSentry';
 import {t} from 'app/locale';
+import PreferencesStore from 'app/stores/preferencesStore';
 import space from 'app/styles/space';
 import {trackAdvancedAnalyticsEvent} from 'app/utils/advancedAnalytics';
 import getCookie from 'app/utils/getCookie';
 
+type Preferences = typeof PreferencesStore.prefs;
+
 export default function DemoHeader() {
   // if the user came from a SaaS org, we should send them back to upgrade when they leave the sandbox
   const saasOrgSlug = getCookie('saas_org_slug');
   const getStartedText = saasOrgSlug ? t('Upgrade Now') : t('Sign Up for Free');
   const getStartedUrl = saasOrgSlug
     ? `https://sentry.io/settings/${saasOrgSlug}/billing/checkout/`
     : 'https://sentry.io/signup/';
+
+  const [collapsed, setCollapsed] = useState(!PreferencesStore.prefs.collapsed);
```

Comment:
@roggenkemper I'm confused why we invert the state for collapsed here

Reply:
you're right. this came from trying to get the onPreferenceChange to just call setCollapse but it isn't needed and I forgot to delete